tests/game/type-advantage.test.ts

```diff
@@ -0,0 +1,72 @@
+import {describe, expect, test} from '@jest/globals'
+import HotguyCommon from 'common/cards/alter-egos-iii/hermits/hotguy-common'
+import PoultrymanCommon from 'common/cards/alter-egos/hermits/poultryman-common'
+import {Hermit} from 'common/cards/base/types'
+import GoodTimesWithScarCommon from 'common/cards/default/hermits/goodtimeswithscar-common'
+import GrianCommon from 'common/cards/default/hermits/grian-common'
+import IJevinCommon from 'common/cards/default/hermits/ijevin-common'
+import ImpulseSVCommon from 'common/cards/default/hermits/impulsesv-common'
+import MumboJumboCommon from 'common/cards/default/hermits/mumbojumbo-common'
+import TinFoilChefCommon from 'common/cards/default/hermits/tinfoilchef-common'
+import XBCraftedCommon from 'common/cards/default/hermits/xbcrafted-common'
+import {RowComponent} from 'common/components'
+import query from 'common/components/query'
+import {attack, endTurn, playCardFromHand, testGame} from './utils'
+
+describe('Type advantage tests', () => {
```

Comment:
It's better to place this test at the bottom of the file. It's generally better to use things after they are defined even when it's not required.